dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task230420AlterVarcharLengthOfLockedByColTest.java

```diff
@@ -0,0 +1,71 @@
+package com.dotmarketing.startup.runonce;
+
+
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.db.DbType;
+import com.dotmarketing.exception.DotDataException;
+import org.junit.Test;
+
+import java.sql.SQLException;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+public class Task230420AlterVarcharLengthOfLockedByColTest {
+    // Change the length to recreate the scenario
+    private void setMinLengthBeforeTask (String tblName) throws SQLException {
+        String query = "";
+        if (DbType.POSTGRESQL == DbType.getDbType(DbConnectionFactory.getDBType())){
```

Comment:
We have this method that checks the DB type
```
if (DbConnectionFactory.isPostgres()) {
```

Reply:
db type method implemented, the final keyword was added in most of the cases where it can be implemented
<img width="722" alt="image" src="https://user-images.githubusercontent.com/127987858/234046317-88d83dbe-1fa2-4aa1-8441-330cb4054aef.png">